examples/multipleContracts/multi.act

```diff
@@ -0,0 +1,29 @@
+behaviour remote of B
+interface set_remote(uint _x)
+
+iff
+   CALLVALUE == 0
+
+storage
+
+  y => 1
+
+// we can specify how a call to a function of B
+// updates the storage of A by labelling the storage header
+storage a
+   x => _x
+
+behaviour create_a of B
+interface create_a()
+
+iff
+   CALLVALUE == 0
+
+storage
+   // newAddr should be a builtin function
+   a => newAddr(ADDRESS, NONCE)
+
+// storage labels can also introduce new contracts
+creates storage newAddr(ADDRESS, NONCE)
```

Comment:
I don't really understand this section.

Reply:
the syntax `storage A` allows us to specify the state changes in the contract A in a spec of B. The syntax `creates storage` allows us to specify the initial state of a contract. This is the combination of the two

Reply:
Is `A` missing there? Or do I still not get it?
`creates storage A newAddr(ADDRESS, NONCE)`

Reply:
Yeah `creates storage A at newAddr(ADDRESS, NONCE)` might be necessary since we need to know the code for `A`.